packages/mpl-point-clicker.yml

```
@@ -1,5 +1,5 @@
 name: mpl_point_clicker
```

Can you make this consistent?
```suggestion
name: mpl-point-clicker
```